### PR TITLE
Moving text practice

### PR DESCRIPTION
**Pull request by Aadya-Jain**
## Location

- Chapter(s): 2
- Section(s): 2.4

A practice problem telling the user to move the text by blitzing it onto a rectangle or moving it with coordinates

### CODE REVIEW

**Review by hpanchal092 — COMMENTED** [review]
Code looks really good and works as intended, 👍. Can you change the both of the file names to something like MovingText.py, and move the solution code into the `./games/chapter2/solution` directory, and move the template to the `./games/chapter2/practice` directory?

**Review by hpanchal092 — APPROVED** [review]
LGTM 👍

File: games/chapter2/solutions/moving-text.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    # draws text onto the rectangle
+    window.blit(text, white_rectangle)
+    # you can also use coordinates in the form of a tuple
+    # the coordinates would place the top left pixel
+    # Syntax: window.blit(text, (x,y))
+
+    # update the screen
+    pygame.display.update()
+
+    syntax: pygame.time.wait(30)  # adds a 30 millisecond delay
```
Comment (chrehall68) [inline games/chapter2/solutions/moving-text.py]: Is this supposed to be commented out? Or why is the `syntax:` there?

Comment (chrehall68) [reply games/chapter2/solutions/moving-text.py]: I dont know what this is, but it still runs whatever statement you have on the right, so it seems fine.

**Review by chrehall68 — APPROVED** [review]

File: games/chapter2/solutions/moving-text.py (hunk 1 above)
Comment (Aadya-Jain) [reply games/chapter2/solutions/moving-text.py]: the syntax: is supposed to show what code they could've used if they used coordinates, should i delete/rephrase it?

Comment (hpanchal092) [reply games/chapter2/solutions/moving-text.py]: we're looking at the one at line 56, because eliot and i have never seen that before.

Comment (Aadya-Jain) [reply games/chapter2/solutions/moving-text.py]: I think that was a typo, sorry!

**Review by chrehall68 — APPROVED** [review]
LGTM. Thanks for changing that confusing line.

File: games/chapter2/practice/moving_text.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# to make it more clear
+
+
+# imports!
+
+SCREEN_SIZE = (800, 800)
+BLACK = (0, 0, 0)
+BLUE = (0, 0, 255)
+WHITE = (255, 255, 255)
+momentum = [2, 2]  # down and right
```
Comment (Chrovo) [inline games/chapter2/practice/moving_text.py:21]: Small nit: the `momentum` variable should probably be a tuple for consistency as you used a tuple in the other file.

**Pull request merged**